tests/ctest/test_opmphm.c

```diff
@@ -155,11 +155,11 @@ static void test_cyclicCountUpEdges (void)
 			exit_if_fail (opmphm, "opmphmNew");
 			OpmphmGraph * graph = opmphmGraphNew (opmphm, rUniPar, n, 1);
 			exit_if_fail (graph, "opmphmGraphNew");
-			OpmphmInit init;
+			OpmphmInit opmphmInit;
 			// dummy data
-			init.getName = test_opmphm_getName;
-			init.initSeed = (int32_t)1;
-			init.data = (void **)1;
+			opmphmInit.getName = test_opmphm_getName;
+			opmphmInit.initSeed = (int32_t)1;
+			opmphmInit.data = (void **)1;
```

Comment:
Why?

Reply:
`elektra-gcc-configure-debian-wheezy` did not like it.
```
/home/jenkins/workspace/workspace/elektra-gcc-configure-debian-wheezy/tests/ctest/test_opmphm.c:119:15: warning: declaration of 'init' shadows a global declaration [-Wshadow]
In file included from /home/jenkins/workspace/workspace/elektra-gcc-configure-debian-wheezy/tests/cframework/tests_internal.h:16:0,
                 from /home/jenkins/workspace/workspace/elektra-gcc-configure-debian-wheezy/tests/ctest/test_opmphm.c:11:
/home/jenkins/workspace/workspace/elektra-gcc-configure-debian-wheezy/tests/cframework/tests.h:44:5: warning: shadowed declaration is here [-Wshadow]
```